Sprint-2/stretch/till.js

```diff
@@ -5,27 +5,29 @@
 // Then it should return the total amount in pounds
 
 function totalTill(till) {
+  
+  const tillCoins = {
```

Comment:
This variable name confused me initially. I think you can probably come up with a less-confusing name or use comment to describe what it represents.